lib/ord.js

```diff
@@ -151,9 +147,9 @@ function createDefaultORDDocument(linkedCsn, appConfig) {
 }
 
 function extractPackageIds(ordDocument) {
-    const packageIds = new Set();
+    const packageIds = [];
     if (ordDocument.packages) {
-        ordDocument.packages.map((pkg) => packageIds.add(pkg.ordId));
+        ordDocument.packages.map((pkg) => packageIds.push(pkg.ordId));
```

Comment:
@zongqichen : I though there is no need for using `Set`; instead, we should use `Array` data-structure.

Reply:
Good point! I will check now.